dotnet/src/Connectors/Connectors.OpenAI/OpenAIPromptExecutionSettings.cs

```diff
@@ -19,16 +19,18 @@ public sealed class OpenAIPromptExecutionSettings : PromptExecutionSettings
     /// <summary>
     /// Temperature controls the randomness of the completion.
     /// The higher the temperature, the more random the completion.
+    /// Default is 1.0.
```

Comment:
Are we sure to put this as full 1.0 for both?

This is full creative mode 😊

> Using a GPT model, setting both Top P (nucleus sampling) and Temperature to 1 can be a good starting point. Top P at 1 means the model will consider a wider range of words for each choice, leading to more diverse responses. Similarly, a Temperature of 1 maintains a balance between randomness and predictability in the responses. You can adjust these settings based on the specific requirements of your application, like more creative versus more predictable responses.

Reply:
Those are the defaults from OpenAI, answers the question.

![image](https://github.com/microsoft/semantic-kernel/assets/19890735/73dda805-1d84-4174-9db9-eb1c3c2a5134)